core/src/test/java/apoc/atomic/AtomicTest.java

```diff
@@ -161,19 +165,31 @@ public void testRemoveLastItemArray(){
 		assertTrue(ages.length == 0);
     }
 
-    @Test(expected = RuntimeException.class)
+    @Test
     public void testRemoveOutOfArrayIndex(){
         db.executeTransactionally("CREATE (p:Person {name:'Tom',age: [40,50,60]})");
-        testCall(db, "MATCH (n:Person {name:'Tom'}) CALL apoc.atomic.remove(n,$property,$position,$times) YIELD container RETURN count(*)",map("property","age","position",5,"times",5), (r) -> {});
-		long[] ages = TestUtil.singleResultFirstColumn(db, "MATCH (n:Person {name:'Tom'}) RETURN n.age as age;");
-		assertThat(ArrayUtils.toObject(ages), Matchers.arrayContaining(40L, 50L, 60L));
+
+		QueryExecutionException e = assertThrows(QueryExecutionException.class,
+				() ->  testCall(db, "MATCH (n:Person {name:'Tom'}) CALL apoc.atomic.remove(n,$property,$position,$times) YIELD container RETURN count(*)",
+						map("property","age","position",5,"times",5), (r) -> {})
+
+		);
+		Throwable except = ExceptionUtils.getRootCause(e);
+        assertTrue(except instanceof RuntimeException);
+		assertEquals("Attention your position out of range or higher than array length, that is 3", except.getMessage());
```

Comment:
Nothing to do with this PR, but maybe we need to fix the English of our APOC Error messages sometime 🙈

Reply:
I fixed this one as it was indeed very bad English